doc/HowMuch.txt

```diff
@@ -20,23 +20,23 @@ This plugin was written and tested under Vim 7.4.
 ==============================================================================
 CONTENTS                                                    *HowMuch-contents*
 
-    1. Intro ............................................. |HowMuch-intro| 
-    2. Features .......................................... |HowMuch-features| 
+    1. Intro ............................................. HowMuch-intro|
+    2. Features .......................................... HowMuch-features|
```

Comment:
I think here we need the first pipe so that the reader can jump to the right position.